.github/workflows/js-release.yml

```diff
@@ -0,0 +1,79 @@
+name: '[js] Release'
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - package.json
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: 'Setup NodeJS 20.x'
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: 'Get version from package.json'
+        run: |
+          VERSION=$(jq -r '.version' package.json)
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "TAG_NAME=Sails-JS/v$VERSION" >> $GITHUB_ENV
+
+      - name: 'Get previous version from npm registry'
+        run: |
+          PREVIOUS_VERSION=$(npm show @js/package version)
+          echo "PREVIOUS_VERSION=$PREVIOUS_VERSION" >> $GITHUB_ENV
+
+      - name: 'Compare versions'
+        run: |
+          if [ ${{ env.VERSION }} == ${{ env.PREVIOUS_VERSION }} ]; then
```

Comment:
It is better off to use step outputs as it is easy to lose setting up some var. For example, now this code is highlighted in VSCode, whereas if it were from steps, it would be highlighted only if you forgot to expose this var as an output